_src/assets/js/collapseform.js

```diff
@@ -8,103 +8,60 @@ const iconDesignElement = document.querySelector('.icon_design');
 const iconFillElement = document.querySelector('.icon_fill');
 const iconShareElement = document.querySelector('.icon_share');
 
-function handleCollapseDesign() {
-  designContentElement.classList.toggle('hidden');
-  if (iconDesignElement.classList.contains('fa-angle-up')) {
-    iconDesignElement.classList.remove('fa-angle-up');
-    iconDesignElement.classList.add('fa-angle-down');
+function toggleCollapsibleIcon(collapsibleIcon) {
+  if (collapsibleIcon.classList.contains('fa-angle-up')) {
+    collapsibleIcon.classList.remove('fa-angle-up');
```

Comment:
Hi @palomaviolin so..you changed the handleCollapseDesign function for toggleCollapsibleIcon but they do the same, right?